guides/kubernetes.adoc

```diff
@@ -130,6 +130,56 @@ kubectl port-forward service/helloworld 8080:8080
 
 and access your application at http://localhost:8080/.
 
+## Deploy and expose the application to the outside world
+
+Now that you are satified with your application you might want to expose it to the outside world.
+The `wildfly` helm chart can alos create an `Ingress` resource for you so that your application is available oustide of the cluster.
+
+### Create a secret for TLS
+
+First you need to create a `secret` to hold the certificates used for TLS connection.
+
+[source,bash]
+----
+kubectl create secret tls helloworld-secret-tls --key="tls.key" --cert="tls.crt"
+----
+
+### Deploy your application
+
+Now we need to edit the previous `helm.yaml` file to provide the required parameters:
+
+
+.helm.yaml
+[source,yaml]
+----
+image:
+  name: quay.io/<YOUR_USERNAME>/helloworld
+build:
+  enabled: false # Disable S2I build
+deploy:
+  imagePullSecrets:
+    - name: <YOUR_USERNAME>-pull-secret
+  route:
+    enabled: false # Disable OpenShift Route
+  ingress:
+    enabled: true #Enable Ingress resource creation
+    host: <YOUR-EXPOSED-DNS-NAME>
+    tls:
+      enabled: true #Enable Ingress TLS
+      secret: helloworld-secret-tls
+----
+You can now deploy your application with these values:
+
+[source,bash]
+----
+helm install helloworld \
```

Comment:
we could use `helm upgrade` instead so that it would override the previous release. wdyt?

Reply:
+1